src/styles/booleanDropdown.scss

```diff
@@ -30,24 +30,52 @@
 }
 
 .is-open {
-  background-color: lightgray;
+  background-color: $dropdown-open;
 }
 
-.arrow {
+.arrow-closed {
   background: url('../assets/arrow.svg') no-repeat right;
   display: inline-block;
   padding: 4px;
 }
 
+.arrow-open {
+  background: url('../assets/arrow.svg') no-repeat right;
+  display: inline-block;
+  padding: 4px;
+  transform: scaleY(-1);
+}
+
```

Comment:
you can have a `.arrow` class <for all overlapping> and a `flipped` class for just the `transform` part